### ISSUE TITLE
RDS headers not found

### ISSUE THREAD

**Issue by mclubb** [aws/aws-sdk-cpp#475]
Hello, 

I have successfully created an example connecting to S3, so I figured I would try to pull some data from RDS. I added the following includes

```
#include <aws/rds/RDSClient.h>
#include <aws/rds/model/DescribeReservedDBInstancesResult.h>
```

and added the code to use these however, when I run make on the command line (not using xcode on mac) I am getting the following error:

```
main.cpp:5:10: fatal error: 'aws/rds/RDSClient.h' file not found
#include <aws/rds/RDSClient.h>
         ^
1 error generated.
make[2]: *** [CMakeFiles/AWS_TEST.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/AWS_TEST.dir/all] Error 2
make: *** [all] Error 2
```

Question is why would it find  
```
#include <aws/core/Aws.h>
#include <aws/s3/S3Client.h>
#include <aws/s3/model/Bucket.h>
```

but not these

```
#include <aws/rds/RDSClient.h>
#include <aws/rds/model/DescribeReservedDBInstancesResult.h>
```

**Comment by MikeClubb** [comment]
Ok, I have gotten a bit further. After re-reading over and over about how to configure the CMakeList.txt, I realized why the s3 header files were found. I needed to add the rds to link libraries. `target_link_libraries(AWS_TEST aws-cpp-sdk-s3 **aws-cpp-sdk-rds**)`

However, I am not getting

`main.cpp:10:10: error: no type named 'SDKOptions' in namespace 'Aws'
    Aws::SDKOptions options;
    ~~~~~^
error: no type named 'InitAPI' in namespace 'Aws'
    Aws::InitAPI(options);
    ~~~~~^
...
`
Any ideas? I will keep poking around until I find out something.
